#!/usr/bin/env python
PACKAGE = "template_ws"

from dynamic_reconfigure.parameter_generator_catkin import *

gen=ParameterGenerator()
gen.add('max_x',double_t,0,"Max_x",0.0,-100.0,100)
gen.add('min_x',double_t,0,"Min_x",-100.0,-100,100)
gen.add('max_y',double_t,0,"Max_y",12.0,-50,50)
gen.add('min_y',double_t,0,"Min_y",-12.0,-50,50)
gen.add('max_z',double_t,0,"Max_z",-0.8,-10,10)
gen.add('min_z',double_t,0,"Min_z",-1.8,-10,10)
gen.add('start_index',int_t,0,"Starting index",0,0,127)
gen.add('end_index',int_t,0,"End index",127,0,127)
gen.add('point_start',int_t,0,"Starting index",0,0,512)
gen.add('point_end',int_t,0,"End index",512,0,2048)

exit(gen.generate(PACKAGE, "template_ws", "template_ws"))